consumer/consumer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/adjust/rmq"
+	"github.com/dghubble/go-twitter/twitter"
+	"github.com/joho/godotenv"
+)
+
+// TaskConsumer implements the Consumer interface
+type TaskConsumer struct {
+}
+
+// Consume is the work that will be done by the taskconsumer
+func (consumer *TaskConsumer) Consume(delivery rmq.Delivery) {
```

Comment:
All capitolized names will be exported, but I don't think you meant to have this exported?